runtime-config.md

```diff
@@ -22,6 +23,13 @@ In Linux, for e.g., the network namespace could be configured in this hook.
 
 If a hook returns a non-zero exit code, then an error including the exit code and the stderr is returned to the caller and the container is torn down.
 
+### Post-start
+
+The post-start hooks are called after the user process is started.
+It can be used for notifying user that real process is spawned.
```

Comment:
On Mon, Sep 28, 2015 at 03:23:36PM -0700, Alexander Morozov wrote:
> > +It can be used for notifying user that real process is spawned.
> 
> I don't like sidecar application example. Why you insist on using
> it?

Because it's concrete and easier to phrase (and that application may
be in a separate container, which may address your concerns).  Do you
have another example of when you'd actually care about the “user
application has started” event?  Just logging it seems like something
that could be handled by an event stream like ‘runc events’, and not
something that you'd need event-specific hooks for.